dist/unix/systemd/qbittorrent-nox@.service.in

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=qBittorrenti-nox service for user %I
+
+Documentation=man:qbittorrent-nox(1)
+After=network.target
+
+[Service]
+User=%i
+ExecStart=@EXPAND_BINDIR@/qbittorrent-nox
+Type=simple
+PrivateTmp=yes
```

Comment:
@Chocobo1:  

>this is the first time I encounter this kind of shorthand... 

Actually, that shorthand is pretty standard command-line syntax for defining file access permissions, and database variables.  

That being said, it does make very little sense to use it in an otherwise conventional English sentence.